app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -7,14 +7,19 @@ import AddressSearch, {
   transformKeys,
   snakeCase,
   LocationQuery,
-  LOCATIONS_URL,
 } from '@18f/identity-address-search';
 import BackButton from './back-button';
 import AnalyticsContext from '../context/analytics';
 import InPersonLocations, { FormattedLocation } from './in-person-locations';
 import { InPersonContext } from '../context';
 import UploadContext from '../context/upload';
 
+export const LOCATIONS_URL = new URL(
+  '/verify/in_person/usps_locations',
+  window.location.href,
+).toString();
+export const ADDRESS_SEARCH_URL = '/api/addresses';
```

Comment:
This is a little inconsistent? One of these is a plain string, the other is  URL object, I have a few suggestions:
- name the string one as `_PATH` and the other as `_URL`
- promote both to `URL` objects?

Reply:
We may not need the `URL` constructor here anymore. It was added in the Node.js v18 upgrade, since the Mocha specs use of `swr` required fully-qualified URLs, since Node's `fetch` implementation can't use paths (unlike how it works in a browser). Since it appears we may not be referencing these constants in the specs anymore (??), they could be removed.